plugins/amoeba/openmmapi/include/openmm/AmoebaMultipoleForce.h

```diff
@@ -326,6 +332,14 @@ class OPENMM_EXPORT_AMOEBA AmoebaMultipoleForce : public Force {
     void getInducedDipoles(Context& context, std::vector<Vec3>& dipoles);
 
     /**
+     * Get the total dipole moments of all particles.
+     *
+     * @param context         the Context for which to get the induced dipoles
+     * @param[out] dipoles    the induced dipole moment of particle i is stored into the i'th element
```

Comment:
The above two lines should say "total" instead of "induced".  To make the function of this method completely clear, it might also be good to change the summary to, "Get the total (fixed plus induced) dipole moments of all particles."